README.md

```diff
@@ -127,8 +127,15 @@ $ git clone https://github.com/pulp/pulplift.git
 - Vagrant provider plugin (follow [vagrant](
   https://www.vagrantup.com/docs/providers/installation.html) instructions)
   - libvirt and virtualbox supported
+- Recommended: [Vagrant sshfs plugin](https://github.com/dustymabe/vagrant-sshfs#install-plugin) if using libvirt
```

Comment:
For the source scenario with the instructions below (shared folder), it's actually required.